### ISSUE TITLE
Migrate "Management SDK retirement guidance" to Eng Hub

### ISSUE THREAD

**Issue by MrJustinB** [Azure/azure-sdk-tools#6720]
https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/856/Management-SDK-retirement-guidance

**Comment by ladonnaq** [comment]
@josefree - Should this information be moved to engineering hub and be under the section for SDK deprecation?  Justin recently rewrote Mario's doc page on SDK deprecation that was still in Azure Devops and moved it to engineering hub.  Here is the SDK deprecation doc - https://eng.ms/docs/products/azure-developer-experience/develop/sdk-deprecation.

**Comment by maririos** [comment]
My recommendation before moving the doc will be for @josefree to work with @mario-guerra and integrate the process so the current SDK retirement guidance doc in eng hub can apply to any type of library, independent if it is data or control plane.
If the user can do the retirement, we shouldn't do it for them

**Comment by ladonnaq** [comment]
> My recommendation before moving the doc will be for @josefree to work with @mario-guerra and integrate the process so the current SDK retirement guidance doc in eng hub can apply to any type of library, independent if it is data or control plane. If the user can do the retirement, we shouldn't do it for them
I was thinking the same thing but was going to confirm with Josephine that what is currently in Azure Devops is still valid.  Then I was going to rewrite and consult Mario.  From looking at the doc and what I understand about the retirement process, there should not be much difference in the process/guidance for management vs data plane.

**Comment by ladonnaq** [comment]
@josefree - Let's discuss this week. I will send you a meeting invite.  As I understand we want to also include APIs in the retirement/deprecation process.  I think we need a top-level category for "Retirement/deprecation" and all related docs will go there.

**Comment by ladonnaq** [comment]
Josephine initiated an email thread to discuss adding a rule to SDK Breaking Change process to require API Specs to be deprecated and archived.  There was never any closure to the email thread (i.e. action items, next steps). I forwarded the email thread to Josephine and copied Mariana. In the email I requested Josephine to determine if the content in ADO is valuable to our service partners. We have detailed information on how to deprecate SDKs on engineering hub already. Also, if the decision is that API Specs must be deprecated (removed from API Specs repo and archived in a different repo), then we need updates to the docs to provide guidance. In addition, we have a requirement in backlog to support API and SDK deprecation. This is something we could define the requirements for this semester and implement next semester.
